index.html

```diff
@@ -171,15 +170,59 @@ <h2>Multi-Post Stories </h2>
                 <button class="button" type="button" onclick="">See Project</button>
             </div>
         </div>
-
-
-
-
-
-
-
-
-
+<footer>
+    <div class="template-footer">
+        <h1>CONTACT US</h1>
+        <p>If you have an application you are <br>
+            interested in developing, a feature that you <br> 
+            need built or a project that needs coding. <br>
+            I’d love to help with it </p>
+
+            <div class="form-section">
+                <form class="form-submit" action="https://formspree.io/f/form_id" method="post">
+                    <ul>
+                        <li>
+                            <label for="name"></label>
+                            <input type="text" id="name" name="user_name" placeholder="text" maxlength="30" required>
+                        </li>
+                        <li>
+                            <label for="email"></label>
+                            <input type="email" id="email" name="email" placeholder="text" required value>
+                        </li>
+                        <li>
+                            <label for="msg"></label>
+                            <textarea name="Enter text here" id="msg" placeholder="text" required cols="30" rows="10" maxlength="500"></textarea>
+                        </li>
+                    </ul>
```

Comment:
- [ ] The form doesn't look like a list so the `<ul>` could be removed in this case for better semantics.
 - [ ] The template doesn't appear to have a label over the form inputs, only a placeholder text so it could be removed.
 - [ ] Kindly add a correct placeholder text for each input so it can be easier for the user to understand what to write on each input box.
 - [ ] The input style looks wider in the Figma template and looks like when input has text or when it's active it becomes slightly blue. It does not look that the second one is blue and the others aren't. Kindly change its style to match the Figma template. Remember to style the `<textarea>` like you did for the other inputs.
 -Your design:
<img width="417" alt="image" src="https://user-images.githubusercontent.com/61919617/186493234-49bdd007-54aa-4b79-867f-84079aca409a.png">
-Figma template:
<img width="395" alt="image" src="https://user-images.githubusercontent.com/61919617/186493286-2f820b97-0088-4cb1-9855-98b1d34dae52.png">